install.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
```

Comment:
Now that you mention it, I don't like that this script mentions a `-dev` version, because it simply won't ever work, because we'll never actually make a `-dev` release.

Maybe the the `replace-version` regions should have three states:
- `stable` (will be set to the latest suffixless version)
- `unstable` (will be set to the latest released version, since `-dev` isn't released, it won't include it, but it will include things like `1.0.0-rc`)
- `dev` (will be set any latest version, including `-dev`)

Reply:
Having three states for this would make a lot of sense. Are you okay with adding the third state? It's totally fine, if you want to be done, then we can merge it as is. :)

Thank you for moving the file! :D